cfme/tests/automate/custom_button/test_custom_menu.py

```diff
@@ -54,4 +79,11 @@ def test_custom_menu_display():
     Bugzilla:
         1678151
     """
-    pass
+    appliance.update_advanced_settings(ADVANCE_SETTING_CUSTOM_MENU)
+    appliance.reboot()
+    view = navigate_to(appliance.server, "LoggedIn")
+
+    for menu in ["RedHat", "ManageIQ"]:
```

Comment:
Thoughts on parametrizing the test case against these two different menus? Probably not worth it due to increased time for the test to run, especially with the reboot. But just curious.

Reply:
Actually `reboot` time is not issue. If you check, we are updating both menu at a same time so no need of parametrization. this loop only checks are similar for both menus.

If we parametrized it will just like dummy test like performing same thing two time.